### ISSUE TITLE
Module support

### ISSUE THREAD

**Issue by tylerFowler** [hashicorp/terraform-plugin-docs#194]
Are there any plans to support documentation generation for Terraform Module codebases in the same fashion as providers?

**Comment by bflad** [comment]
Hi @tylerFowler 👋 Thank you for raising this!

The design scope of this tool was _intended_ to be for provider documentation only. That being said, I have added this to our team's next triage meeting to discuss this further just to ensure we don't want to change that stance.

There are some existing community tools that can generate Terraform module documentation, such as https://terraform-docs.io/, which has been around for quite some time now and is well vetted in the module ecosystem.

**Comment by tylerFowler** [comment]
Indeed, I was surprised that this tool _didn't_ cover that scope after seeing a lot of nice, clearly auto-generated docs for modules in the official registry - so the scope made sense to me at first glance. However if the official recommendation is to use terraform-docs that's also fine with me, it seems like a quality tool as well.
